### ISSUE TITLE
CI Nightly Failing

### ISSUE THREAD

**Issue by anirudh2** [graphprotocol/AllocationOpt.jl#20]
**Describe the bug**
See https://github.com/graphprotocol/AllocationOpt.jl/actions/runs/3277996015/jobs/5395895988.
We get the error:
```
ERROR: Could not find either `git-tree-sha1` or `path` for package DelimitedFiles
```
This is likely because the nightly version of julia is 1.9, whereas we only officially support 1.7.

**Comment by anirudh2** [comment]
Proposed fix: @hopeyen, since we only support 1.7 officially, I would recommend removing the CI nightly test. Thoughts?

**Comment by hopeyen** [comment]
No objection, I can remove it!

**Comment by anirudh2** [comment]
Thanks! Feel free to close once passing.

**Comment by hopeyen** [comment]
resolved in:  7d38992179e931745a86b8cbf98bcc071cf684f6 followed by 9ed57c0389104a65d9492daf23a183a28606e56e
